core/src/main/java/com/linecorp/armeria/server/decoding/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+/**
+ * HTTP content decoding service.
+ */
+@NonNullByDefault
+package com.linecorp.armeria.server.decoding;
```

Comment:
How about merging this package into `server.encoding`?

Reply:
move complete! Now I think I know a little about the package policy in armeria 😊

Reply:
Awesome. 😁